07-uxdi-collaboration/ajax/readme.md

```diff
@@ -9,48 +9,36 @@
 - Perform POST, PUT, and DELETE requests to an API to modify data.
 
 ## Opening Framing (5/5)
-In the past couple weeks or so, we've been learning ALOT about server side requests and responses. Currently we know how to create applications with full CRUD on models in our database, and that's great. When we do that CRUD, however, it requires a page reload of some kind. It would be really nice if we could execute some kind of functionality on the client side of our application but still communicate with the backend without a page refresh. If only we had a client side language that was non blocking and asynchronous.
+In the past couple weeks or so, we've been learning a lot about server-side requests and responses. Currently we know how to create applications with full CRUD on models in our database, and that's great. When we do that CRUD, however, it requires a page reload of some kind. It would be really nice if had some functionality on the client side of our application but still communicate with the backend without a page refresh. If only we had a client side language that was non blocking and asynchronous...
 
 (ST-WG) Think back to the first couple weeks of class, whats the difference between synchronous and asynchronous program execution?
```

Comment:
Should we phrase this as why asynchronous behavior is so important here (vs. what's the difference between sync and async)?